### PR TITLE
Added link to the HttpTelemetry Event Source

### PR DESCRIPTION
**Pull request by danespinosa**
## Summary

Adds link to the Event Source for `System.Net.Http`, `System.Net.Security`, `System.Net.NameResolution` and `System.Net.Sockets` to keep consistency across the different Event Sources.

Fixes #Issue_Number (if available)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/diagnostics/available-counters.md](https://github.com/dotnet/docs/blob/52abe7e80da995d4dd1583494c4fb8ec008b79c1/docs/core/diagnostics/available-counters.md) | [Well-known EventCounters in .NET](https://review.learn.microsoft.com/en-us/dotnet/core/diagnostics/available-counters?branch=pr-en-us-41571) |

### CODE REVIEW

File: docs/core/diagnostics/available-counters.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ The following counters are published as part of the [ASP.NET Core Kestrel web se
 
 ## System.Net.Http counters
 
-The following counters are published by the HTTP stack.
+The following counters are published by the HTTP stack and are mantained in  [`HttpTelemetry.AnyOS.cs`](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.AnyOS.cs).
```
Comment (IEvangelist) [inline docs/core/diagnostics/available-counters.md:86]:
```suggestion
The following counters are published by the HTTP stack and are maintained in [_HttpTelemetry.AnyOS.cs_](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.AnyOS.cs).
```

**Review by IEvangelist — APPROVED** [review]

File: docs/core/diagnostics/available-counters.md (hunk 1 above)
Comment (danespinosa) [reply docs/core/diagnostics/available-counters.md:86]: Thanks @IEvangelist should I update all the other links for consistency?

Comment (IEvangelist) [reply docs/core/diagnostics/available-counters.md:86]: Yeah, whenever we refer to source files for example, they should be italicized. And inline code doesn't actually get rendered as such when it's a link.

**Comment by danespinosa** [comment]
@IEvangelist  I made the proposed changes on all links and also added 3 more for System.Net.Security, System.Net.NameResolution and System.Net.Sockets

**Pull request merged**